indigo/release-armhf-build.yaml

```diff
@@ -2,41 +2,16 @@
 # ROS buildfarm release-build file
 ---
 abi_incompatibility_assumed: true
-jenkins_binary_job_priority: 82
+jenkins_binary_job_priority: 83
 jenkins_binary_job_timeout: 120
```

Comment:
The armhf need noteably longer timeouts. Manually testing rviz took 6 hours. Lets try 480, hopefully rviz is the slowest.